### PR TITLE
Add a badge that counts honks

### PR DESCRIPTION
**Pull request by tomayac**
As discussed in internal team chat, this PR adds the ability to count the number of honks:

Screen Shot 2019-08-13 at 13 43 57

### CODE REVIEW

File: app/scripts/main.min.js [hunk 1]
```diff
@@ -165,12 +165,19 @@ const Installer = function(root) {
   root.addEventListener('touchend', install.bind(this));
 };
 
-const AirHorn = function(root) {
+const AirHorn = function(root, isInstalled) {
```
Comment (PaulKinlan) [inline app/scripts/main.min.js]: Why not just have this calculated inline in this function rather than passed in?

Comment (tomayac) [reply app/scripts/main.min.js]: Fair. I did something else before, and the passed in variable is an artifact. https://github.com/GoogleChromeLabs/airhorn/pull/50/commits/3bef1cf89406a27917a6ab224f661404c3d0f72c now inlines it.

**Comment by tomayac** [comment]
@PaulKinlan Is this good to be merged?

**Comment by tomayac** [comment]
Let's all [celebrate](https://twitter.com/tomayac/status/1164071995610402817)! `#Impact`

**Pull request merged**